src/Joblify.Core.Api/Offers/OfferController.cs

```diff
@@ -9,24 +10,57 @@ namespace Joblify.Core.Api.Offers
     [Route("api/offer")]
     public class OfferController : Controller
     {
-        private readonly IOfferService _offerService;
+        private readonly IOfferSearchIndex _offerSearchIndex;
```

Comment:
As I mentioned earlier, I think this should be left like it was before - our source of truth is (or at least should be) db, not index after all